### PR TITLE
Added XE8 features: HTML pagination, view website, and voice commands.

### PR DESCRIPTION
**Pull request by allevato**

### CODE REVIEW

File: src/main/java/com/google/glassware/NotifyServlet.java [hunk 1]
```diff
@@ -132,10 +135,41 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
...
+
+  /**
+   * Wraps some HTML content in article/section tags and adds a footer
+   * identifying the card as originating from the Java Quick Start.
+   *
+   * @param content the HTML content to wrap
+   * @return the wrapped HTML content
+   */
+  private static String makeHtmlForCard(String content) {
+    return "<article><section>" + content + "</section>"
```
Comment (alainv) [inline src/main/java/com/google/glassware/NotifyServlet.java]: You should add the auto-paginate class in case the note is really long.

Comment (allevato) [inline src/main/java/com/google/glassware/NotifyServlet.java]: Good idea, done.

**Comment by alainv** [comment]
This LGTM but assigning to mimming to verify as she's the original owner.

File: src/main/java/com/google/glassware/MainServlet.java [hunk 2]
```diff
@@ -68,8 +69,21 @@ public void onFailure(GoogleJsonError error, HttpHeaders headers) throws IOExcep
   }
 
   private static final Logger LOG = Logger.getLogger(MainServlet.class.getSimpleName());
+  public static final String CONTACT_ID = "com.google.glassware.contact.java-quick-start";
```
Comment (mimming) [inline src/main/java/com/google/glassware/MainServlet.java:72]: Do contact IDs need to be globally unique? If not why add so much namespace?

File: src/main/java/com/google/glassware/MainServlet.java [hunk 3]
```diff
@@ -68,8 +69,21 @@ public void onFailure(GoogleJsonError error, HttpHeaders headers) throws IOExcep
...
   public static final String CONTACT_NAME = "Java Quick Start";
 
+  private static final String PAGINATED_HTML =
+      "<article class='auto-paginate'>"
+      + "<h2 class='blue text-large'>Did you know...?</h2>"
+      + "<p>Cats are <em class='yellow'>solar-powered.</em> The time they spend napping in "
```
Comment (mimming) [inline src/main/java/com/google/glassware/MainServlet.java:78]: This cat fact is awesome.

File: src/main/java/com/google/glassware/NotifyServlet.java [hunk 4]
```diff
@@ -132,10 +139,36 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
...
   }
+
+  /**
+   * Wraps some HTML content in article/section tags and adds a footer
+   * identifying the card as originating from the Java Quick Start.
+   *
+   * @param content the HTML content to wrap
+   * @return the wrapped HTML content
+   */
+  private static String makeHtmlForCard(String content) {
```
Comment (mimming) [inline src/main/java/com/google/glassware/NotifyServlet.java:170]: I probably wouldn't have extracted this function for the sake of easier code scan-ability, but I don't feel too strongly about it.

File: src/main/java/com/google/glassware/MainServlet.java (hunk 2 above)
Comment (allevato) [inline src/main/java/com/google/glassware/MainServlet.java:72]: Wasn't sure and it seemed like a good idea at the time, but it looks like they're not after all, so I can shorten it.

File: src/main/java/com/google/glassware/NotifyServlet.java (hunk 4 above)
Comment (allevato) [inline src/main/java/com/google/glassware/NotifyServlet.java:170]: I had planned to use it to make some of the other cards share the same identifying information, but never did yet.

File: src/main/java/com/google/glassware/MainServlet.java (hunk 3 above)
Comment (allevato) [inline src/main/java/com/google/glassware/MainServlet.java:78]: :smiley_cat:

**Pull request merged**